### PR TITLE
Changed Listen Port

### PR DESCRIPTION
**Pull request by h3x3ncr7pt**
**We** want to make that change because we think in that line `app.listen(8080);` maybe a good solution for this one

### CODE REVIEW

**Review by h3x3ncr7pt — COMMENTED** [review]
that would cause a lot of problems ... because `app.listen(80)` is nice also but needs a repair...

that's what we gonna do here : 

- [ ] Try to Include files in the directory of src
- [x] Help other teams find out what just happened
- [ ] Then we may commit your changes

and then :

1. Keep **Looking at this one **
**image**

2. try to fix it out 
3. Commit !

After All of that : 

- Find a job 
- Take responsibility for your jobs
- Quit  !, thanks to @h3x3ncr7pt

File: app.js [hunk 1]
```diff
@@ -4,4 +4,6 @@ var app = express();
 var routes = require('../routes/index');
 app.use('/v1',routes);
 //this would be a lot better than early one
+//Added body-parser too 
+app.use(body-parser);
```
Comment (h3x3ncr7pt) [inline app.js:8]: this won't work ...

Comment (astuxnet) [reply app.js:8]: You're not correct !

File: app.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 var express=require('express');
 var app = express();
+//i am gonna change this one to another bbc arabic...
 var routes = require('../routes/index');
 app.use('/v1',routes);
```
Comment (astuxnet) [inline app.js:5]: this may not work also

**Review by astuxnet — COMMENTED** [review]
checkout these changes and tell me what happened

File: app.js (hunk 2 above)
Comment (h3x3ncr7pt) [reply app.js:5]: it will work

**Pull request merged**